apps/src/gamelab/GameLabP5.js

```diff
@@ -767,12 +768,13 @@ GameLabP5.prototype.preloadAnimations = function (animationList) {
   this.p5.projectAnimations = {};
   animationList.orderedKeys.forEach(key => {
     const props = animationList.propsByKey[key];
+    const frameCount = studioApp.reduxStore.getState().pageConstants.allAnimationsSingleFrame ? 1 : props.frameCount;
```

Comment:
Best practices (from Paul's proposal) - let's create a selector function and use it here.